Resources/Locale/en-US/emotes/emotes.ftl

```diff
@@ -1,2 +1,7 @@
 ﻿emote-deathgasp = seizes up and falls limp, {POSS-ADJ($entity)} eyes dead and lifeless...
 silicon-emote-deathgasp =seizes up and falls limp...
+
+# for IPC
+chat-emote-msg-deathgasp-silicon =
+    With a hiss of grinding servos and a screech of dying myomers,
+    {POSS-ADJ($entity)} suddenly goes silent.
```

Comment:
> With a hiss of grinding servos and a screech of dying myomers, ***his/her/their*** suddenly goes silent

Reply:
Ah fuck. Guh. Where's the list of these possible options so I can read it and know how to not make this mistake again?

Reply:
ctrl-shift-f FuncPossAdj will bring you to a robust toolbox class where those functions are defined

Reply:
Excellent, thank you.